### ISSUE TITLE
[Question] Changing the color of crossings

### ISSUE THREAD

**Issue by LekYang** [eclipse-sumo/sumo#14079]
Just a simple question, is there any method to change the color of the road crossing?
<img width="566" alt="image" src="https://github.com/eclipse-sumo/sumo/assets/52774722/dcff5806-e20e-4529-a6e2-b3afbecf1ab4">

**Comment by m-kro** [comment]
No. There are a few items like road surface markings which cannot be customised. This is not a big thing but has not gotten much attention yet.
